DemoPage/examples/Button/index.js

```diff
@@ -16,20 +16,20 @@ module.exports = React.createClass({
 
       <div className="DemoPage__group">
         <div className="DemoPage__example--button">
-          <Button id="get-started" kind='cta' label='Get Started' icon='chevron-right'/>
+          <Button id="get-started" kind="cta" label="Get Started" icon="chevron-right"/>
```

Comment:
Consistency. Attributes should use double quotes, strings should use single quotes.